docs/tools/dotnet-Commands.md

```diff
@@ -19,9 +19,10 @@ For complete information on `dotnet`, see [.NET Core command-line interface (CLI
 - [**dotnet remove package**](/dotnet/core/tools/dotnet-remove-package): Removes a package reference from the project file.
 - [**dotnet restore**](/dotnet/core/tools/dotnet-restore?tabs=netcore2x): Restores the dependencies and tools of a project. As of NuGet 4.0, this runs the same code as `nuget restore`.
 - [**dotnet nuget locals**](/dotnet/core/tools/dotnet-nuget-locals): Lists locations of the *global-packages*, *http-cache*, and *temp* folders and clears the contents of those folders.
+- [**dotnet new nugetconfig**](/dotnet/core/tools/dotnet-new): Creates a [`nuget.config`](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file) file to configure NuGet's behavior.
```

Comment:
link to nuget config docs needs to be fixed.

Reply:
Sorry about that! Fixed now